packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -185,7 +185,7 @@ export default [
         await createBackupNgJob({
           name: state.name,
           mode: state.isDelta ? 'delta' : 'full',
-          compression: state.compression ? 'native' : '',
+          compression: state.isFull && state.compression ? 'native' : '',
```

Comment:
Why this change?

Reply:
to not keep the compression value when the delta mode is activated after the full mode

Reply:
Why is it needed?

Reply:
It's not needed but its just a detail for keeping the compression default value to false